src/components/molecules/UiLoader/UiLoader.vue

```diff
@@ -81,12 +67,18 @@ const props = defineProps({
     default: 'spinner',
   },
   /**
-   * Use this props to pass attributes to internal child components
+   * Use this props to set transition type.
    */
-  loaderAttrs: {
-    type: Object as PropsAttrs,
-    default: () => ({
-    }),
+  transitionType: {
```

Comment:
@ArkadiuszKrzymienski I change the name to `transition-type`. `mode` is from `Transition` 👍